pom.xml

```diff
@@ -59,6 +60,35 @@
 			</plugin>
 		</plugins>
 	</build>
+	<!--
+We need to enable this profile for now in order to override the imports execution that removes any dependencies from google.
+
+As of now, the imports activation of spring-cloud-build is not working properly. It activates regardless of the inclusion of the .google file on the source folder.
+-->
+	<profiles>
```

Comment:
Nope, it won't work. I tried that. If its on the firehose pom, build will work on the local module, but will fail from the parent pom